_data/teams.yml

```diff
@@ -303,285 +303,274 @@
+        cycling, backpacking, or playing ice hockey.
+      image:       greg-novak
+      linkedin:    gsnovak
+      github:
+      twitter:     GSNovak
+      instagram:
+      flickr:
+      links:
+
+    - name:        Kim 
+      lastname:    Larsen
+      location:    San Francisco
+      division:
+      description: Originally from Denmark, Kim is passionate about bridging the gap between data analysis and business applications. He believes that if an analysis or algorithm cannot be explained at a high level, it is probably too convoluted. He is also a firm believer in Bayesian regression and loves programming (mostly in R these days).
+      image:       kim-larsen
+      linkedin:    https://www.linkedin.com/pub/kim-larsen/3/4a0/3a2 
+      github:
+      twitter:     
+      instagram:
+      flickr:
+      links:   
+     
+    - name:        Tara 
+      lastname:    Lydiard-Martin
+      location:    San Francisco
+      division:
+      description: Tara loves using data and visualizations to make complex systems feel intuitive. In her free time she enjoys ultimate frisbee, dancing, and the Brazilian martial art of capoeira. Hails from Seattle.
+      image:       tara-lydiard-martin
+      linkedin:    taralydiardmartin
+      github:
+      twitter:     lydiardmartin
+      instagram:
+      flickr:
+      links:
+
+    - name:        Zhou
+      lastname:    Yu
+      location:    San Francisco
+      division:
+      description: Grew up in southwest of China, Zhou loves Asian food, especially spicy dishes. He's always trying to apply statistics and machine learning to solving impactful business problem. In his spare time, he enjoys reading and playing basketball and pool. 
+      image:       zhou-yu 
+      linkedin:    zhouyustanford 
       github:
       twitter:
       instagram:
       flickr:
       links:
-        # links: accepts a list of "type" (e.g. "blog") and "url". Omit entirely if not using.
+
+    - name:        John 
+      lastname:    McDonnel
+      location:    San Francisco
+      division:
+      description: John has a passion for learning, often with the aid of machines. His obsessions with English-style tea, bridge, and tweed jackets make him the world's oldest 20-something. He is a devout Bayesian but a Python vs R agnostic. He claims to be from Seattle, but he's lived all over the northerly parts of North America.     
+      image:       john-mcdonnel
+      linkedin:    www.linkedin.com/pub/john-mcdonnell/33/332/658
+      github:      https://github.com/johnmcdonnell
+      twitter:     johnvmcdonnell
+      instagram:
+      flickr:
+      links:
+
+    - name:        Richie 
+      lastname:    Li 
+      location:    San Francisco
+      division:
+      description: Richie likes machine learning, crunching data and writing Java code. He enjoys learning new technologies and tools. During his free time he loves cooking, video gaming and hiking. Hails from the south west of China.
```

Comment:
`south west` is spelled `southwest` at [line 542](https://github.com/stitchfix/stitchfix.github.io/pull/53/files#diff-4dfacf056472ef2c8c2fd870b10c632bR542).  Why don't we pick one spelling for both cases? Thanks!